kartograf/rpki/fetch.py

```diff
@@ -106,26 +105,29 @@ def process_file(file):
                                  "-P",
                                  context.epoch,
                                  ] + tal_options +
-                                 ["-f", file],  # -f has to be last
+                                 ["-f"] + batch,  # -f has to be last
                                  capture_output=True,
                                  check=False)
 
         if result.stderr and context.debug_log:
             stderr_output = result.stderr.decode()
             with debug_file_lock:
                 with open(context.debug_log, 'a') as logs:
-                    logs.write(f'\nfile: {file}\n')
                     logs.write(stderr_output)
         return result.stdout
 
-    with ThreadPoolExecutor() as executor:
-        results = list(tqdm(executor.map(process_file, files), total=len(files)))
-
-    json_results = [result.decode() for result in results if result]
+    total = len(files)
+    batch_size = 250
```

Comment:
I should test more batch size values. 250 worked out better than 100 and 500, so I stuck with it.